### PR TITLE
ORC-686: Upgrade aircompressor to 0.19 and fix UT

### PR DESCRIPTION
**Pull request by williamhyun**
### What changes were proposed in this pull request?
This PR aims to upgrade aircompressor to 0.19 and fix UT. 

### Why are the changes needed?
This will bring the latest changes. 

### How was this patch tested?
Pass the Cis.

### CODE REVIEW

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you for picking up ORC-686.
I filed it with 0.17 and didn't notice that new air compressor version is available.

**Comment by dongjoon-hyun** [comment]
Feel free to merge as a committer.

**Comment by williamhyun** [comment]
Thank you, I will merge this.

**Comment by williamhyun** [comment]
I merged this via https://github.com/apache/orc/commit/2d4daab164d30a637d18cc4fe298d9862c0da27b at GitBox. I still cannot see merge button here.

**Comment by dongjoon-hyun** [comment]
Thank you, @williamhyun . I backported this to branch-1.6, too.

File: java/core/src/test/org/apache/orc/TestVectorOrcFile.java [hunk 1]
```diff
@@ -2264,7 +2264,7 @@ public void testZstd() throws Exception {
         OrcFile.writerOptions(conf)
             .setSchema(schema)
             .compress(CompressionKind.ZSTD)
-            .bufferSize(1000)
+            .bufferSize(2000)
```
Comment (dongjoon-hyun) [inline java/core/src/test/org/apache/orc/TestVectorOrcFile.java:2267]: Hi, @williamhyun . I'll revive this test case. It turns out that this is a valid test coverage.

**Pull request closed without merging**